### ISSUE TITLE
Activate and deactivate a toggle button programmatically.

### ISSUE THREAD

**Issue by lcalisto** [rdub80/aframe-gui#17]
To activate a button the following code will not work:

```
document.getElementById('buttonID').components['gui-button'].setActiveState(true);
```
Nevertheless to deactivate the following code works:

```
document.getElementById('buttonID').components['gui-button'].setActiveState(false);
```
The **setActiveState(false)** deactivates the button but also turns the toggle button mode into a off (non toggle button). What's the best way to programmatically activate and deactivate a button (toggle or non toggle button)? 

In the following example the buttons work fine: https://aframe-gui.herokuapp.com/demo.html but not using: https://rdub80.github.io/aframe-gui/examples/casestudy_amper.html

**Comment by lmalave** [comment]
This is a bug in the button toggle behavior.   I submitted a pull request so there should be an an updated release shortly with this fix.

**Comment by lmalave** [comment]
This has been fixed as of version 0.3.3
